### ISSUE TITLE
About 2D mIOU for joint training

### ISSUE THREAD

**Issue by chaolongy** [wbhu/BPNet#24]
I reproduced your source code with no other substantial changes, and with the parameter settings of UNet34 for the 2D network and Mink18A for the 3D network and voxel size=5cm, the 3D mIOU basically achieves the performance of 70.6 reported in your paper. However, the 2D mIOU is only 47.3, while your paper's result is 65.1, is this something I'm missing？
![屏幕截图 2022-07-24 204435](https://user-images.githubusercontent.com/31812804/180647637-aafe275f-afb5-4411-9989-7cb2bf753f57.png)

**Comment by Holycomfort** [comment]
I met with similar problem. I loaded the pretrained model provided by this project and tested the performance of BPNet with config/scannet/bpnet_5cm.yaml. The mIoU 2D is only about 57, while 3D is about 68.

**Comment by yhyang-myron** [comment]
@chaolongy @Holycomfort Hi, do you use the 2d label from _2d-label-filt.zip file? Why did I only get 26 miou for 2D with the pretrained model? But I got 70.6 miou for 3D just like the performance in the paper.
